Game/Thing.pde

```diff
@@ -0,0 +1,15 @@
+class Thing {
+
+  public PShape shape;
+  public int p, r;
+
+  public Thing() {
+    shape = new PShape();
+  }
+
+  public Thing(PShape s, int p, int r) {
+    this.shape = s;
+    this.p = p;
+    this.r = r;
+  }
+}
```

Comment:
Would need size_p and size_r parameters if we accept rectangles.
The 1st constructor probably doesn't need to initialize PShape ?

Reply:
yes we can add those for the "Things" that have a size bigger than 1

Reply:
And the class needs badly to be renamed! :)